REFERENCE.md

```diff
@@ -0,0 +1,130 @@
+# Reference
+
+<!-- DO NOT EDIT: This document was generated by Puppet Strings -->
+
+## Table of Contents
+
+### Classes
+
+* [`profile_mysql_server`](#profile_mysql_server): Manage a MySQL/MariaDB server configuration.
+
+## Classes
+
+### <a name="profile_mysql_server"></a>`profile_mysql_server`
+
+Should the mysql_home be created?
+
+Raw data to define mysql::db resources.
+
+The GID for the mysql user.
+
+The name of the mysql group.
+
+The home dir of the mysql user.
+
+Location of MySQL log files.
+
+Location of directory that contains the mysql_home.
+NOTE: Might be useful for this to be an array so multiple nested
+dirs can be created as dependencies.
+
+Location of run directory (generally a sub-directory of /var/run/).
+
+UID of the mysql user.
+
+Username of the mysql user.
+
```

Comment:
The docstrings that generate this content are messed up. I think the comments after `@param` lines need to be indented (as already noted in one of the class files).

Reply:
Fixed.